Une erreur s'est produite pendant {{action}} des dÃ©pendances python pour {{skill}}.
